lib/index.js

```diff
@@ -12,19 +12,20 @@ const githubRunNumber = process.env.GITHUB_RUN_NUMBER;
 const githubSHA = process.env.GITHUB_SHA;
 
 async function getCommitData() {
+  const prettyArg = `--pretty=format:${'%H %an %s'.split(' ').join('%n')}`;
   const log = await git()
     .raw([
       'log',
       '-1',
-      '--pretty=format:{%n  "commit": "%H",%n  "author": "%an",%n  "author_email": "%ae",%n  "date": "%ad",%n  "message": "%s"%n}',
-    ], (err, res) => res);
-  const commit = JSON.parse(log);
+      prettyArg,
+    ]);
+  const [sha, author, message] = log.split('\n');
 
   return {
-    author: commit.author,
-    message: commit.message,
-    sha: commit.commit,
-    url: `https://github.com/${githubRepo}/commit/${commit.commit}`,
+    author,
+    message,
+    sha,
+    url: `https://github.com/${process.env.GITHUB_REPOSITORY}/commit/${sha}`,
```

Comment:
The value for `githubRepo` is assigned in the root level of the file. This is causing problems in the tests, because the value is assigned before we can set `process.env.GITHUB_REPOSITORY`. As as simple solution I moved the `process.env.GITHUB_REPOSITORY` in here so we can set the value right before calling the function.